src/ProvidedTypes.fs

```diff
@@ -14740,10 +14743,12 @@ namespace ProviderImplementation.ProvidedTypes
                     match mT.GetInvokeCode with
                     | Some _ when methodBaseT.DeclaringType.IsInterface ->
                         failwith "The provided type definition is an interface; therefore, it should not define an implementation for its members."
+                    (* NOTE: These checks appear to fail for generative abstract and virtual methods.

```

Comment:
These checks we added last year do not appear to work correctly for generated, virtual members.